Makefile

```diff
@@ -195,7 +203,7 @@ validate: build-container
 	$(CONTAINER_RUN) hack/make.sh validate-git-marks validate-gofmt validate-lint validate-vet
 
 # This target is only intended for development, e.g. executing it from an IDE. Use (make test) for CI or pre-release testing.
-test-all-local: validate-local test-unit-local
+test-all-local: validate-local test-unit-local test-integration-local test-system-local
```

Comment:
Starting years-old OpenShift instances, several registry servers, and the other things `test-integration` and `test-system` do, seem rather undesirable to do directly on the developer’s workstation.

(Up to you if you think the target should be renamed.)

Reply:
Oh okay, I reads the "fort CI testing" part of the comment and got excited :grin:  I'll put it back.